src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_drawing_2016_11_diagram.g.cs

```diff
@@ -16,7 +16,7 @@ namespace DocumentFormat.OpenXml.Office2019.Drawing.Diagram11
 {
     /// <summary>
     /// <para>Defines the NumberDiagramInfoList Class.</para>
-    /// <para>This class is available in Office 2007 or above.</para>
+    /// <para>This class is available in Office 2019 or above.</para>
```

Comment:
in the interop documentation we normally use "is supported in 20xx and later" or use "is not supported in 20xx or later". So if supported, use "and later", if not supported, use "or later".

Reply:
Seems like the meaning is the same but I'll defer to others' opinions. I don't see a really strong need except to match Open Specs docs exact wording. Am I missing anything else?

Reply:
I saw that you used the usual wording in other files, so would suggest to make this the same.

Reply:
This is auto generated and something I actually just put in. We can definitely change it to match standard wordage

Reply:
we agreed to make this change in master first and then merge it to the Office2019 project.